bevm/bevmclient/README.md

```diff
@@ -0,0 +1,40 @@
+Navigation: [DEDIS](https://github.com/dedis/doc/tree/master/README.md) ::
+[Cothority](../README.md) ::
+[Building Blocks](../doc/BuildingBlocks.md) ::
+[BEvm](https://github.com/dedis/cothority/blob/master/bevm/README.md) ::
+bevmclient
+
+# bevmclient - CLI tool to deploy and interact with BEvm contracts
+
+For the details on all the options and arguments, invoke the tool using the `--help` option.
+
+## Create a BEvm account
```

Comment:
For all the ## headers, I think it might be better to use gerunds of the verbs, i.e., creating a BEvm account.